es6.md

```diff
@@ -145,3 +145,49 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ImportDeclaration
+
+**FIXME:** Esprima's behavior differs from Acorn's and SpiderMonkey's behavior in that `source` is of type `ModuleSpecifier`, but is otherwise a `Literal`.
```

Comment:
That's why it says `FIXME`. It's something that needs fixed. I'm fine holding onto this until there can be some sort of conclusive agreement. Note that [there is a similar discrepancy over ArrowExpression](https://github.com/estree/estree/blob/master/es6.md#arrowexpression) already committed, with a similar tag. It's something that needs to be worked out.

I just filed an Esprima bug over this: jquery/esprima#1077